plugins/aea-ledger-ethereum/aea_ledger_ethereum/ethereum.py

```diff
@@ -971,21 +976,20 @@ def try_get_gas_pricing(
         finally:
             self._api.eth.set_gas_price_strategy(prior_strategy)  # pragma: nocover
 
-        if old_tip is not None and gas_price_strategy is EIP1559:
-            base_fee_per_gas = (
-                gas_price["maxFeePerGas"] - gas_price["maxPriorityFeePerGas"]
-            )
-            updated_max_priority_fee_per_gas = old_tip * TIP_INCREASE
-            updated_max_fee_per_gas = (
-                updated_max_priority_fee_per_gas + base_fee_per_gas
-            )
```

Comment:
The second issue with the repricing was that we were not updating both fees by 10%. This was fixed in 1193c13faae2696fb35e2b1ee7a412c78432bf0e.

Reply:
In the beginning, I thought that only the `maxPriorityFeePerGas` had to be increased by at least 10%. However, when testing this approach, I was getting an error message indicating that either `maxPriorityFeePerGas` or `maxFeePerGas` had to be at least `n`, where `n` was a 10% increase of the `maxFeePerGas`. Therefore, I concluded that we should increase both by at least 10%.